ROADMAP.md

```diff
@@ -44,45 +44,72 @@ Any dates listed below and the specific issues that will ship in a given milesto
+  * Cache / Redis [#137](https://github.com/crossplaneio/crossplane/issues/137), [#282](https://github.com/crossplaneio/crossplane/issues/282)
+  * Buckets [#295](https://github.com/crossplaneio/crossplane/issues/295), [#109](https://github.com/crossplaneio/crossplane/issues/109)
+
 * Performance and Efficiency
   * Reconciliation requeue pattern [#241](https://github.com/crossplaneio/crossplane/issues/241)
+
+* UX Enhancements
+  * Enhanced kubectl printer columns [#38](https://github.com/crossplaneio/crossplane/issues/38)
+
 * Engineering
   * General resource controller used for more types [#276](https://github.com/crossplaneio/crossplane/issues/276)
-  * Common code refactoring [#83](https://github.com/crossplaneio/crossplane/issues/83)
+  * Controllers use consistent logging [#7](https://github.com/crossplaneio/crossplane/issues/7)
 
-## v0.3 - Run Real-world Applications
+## v0.3 - Enable Partners to Build Infra Stacks
 
 * Support for a few real-world applications on-top of Crossplane
   * GitLab [#284](https://github.com/crossplaneio/crossplane/issues/284)
   * More applications to follow
+
+* Resource class enhancements: default classes, validation, annotation
+  * Default resource classes - increases claim portability [#151](https://github.com/crossplaneio/crossplane/issues/151)
+  * Resource classes can be validated and annotated [#613](https://github.com/crossplaneio/crossplane/issues/613)
+
+* Infra stacks (out-of-tree) with single-region secure connectivity between k8s and DBaaS, Redis, Buckets 
+  * Stacks Manager: app vs. infra stacks, namespace isolation, annotation support [#609](https://github.com/crossplaneio/crossplane/issues/609)
+  * Move infra stacks (GCP, AWS, Azure) into separate repos & upgrade to kubebuilder2 [#612](https://github.com/crossplaneio/crossplane/issues/612)
+  * GCP infra stack: single-region secure connectivity: GKE & CloudSQL, CloudMemorystore, Buckets [#615](https://github.com/crossplaneio/crossplane/issues/615)
+  * AWS infra stack: single-region secure connectivity: EKS & RDS, ElastiCache, Buckets [#616](https://github.com/crossplaneio/crossplane/issues/616)
+  * Azure infra stack: single-region secure connectivity: AKS & AzureSQL, AzureCache, Buckets[#617](https://github.com/crossplaneio/crossplane/issues/617)
+  * Stacks v1 CLI / kubectl plugin: init, build, push commands [#614](https://github.com/crossplaneio/crossplane/issues/614)
+* Docs & examples
+  * Infra stack developer guide [#610](https://github.com/crossplaneio/crossplane/issues/610)
+  * Portable wordpress app stack (kubebuilder-based) published to registry [#572](https://github.com/crossplaneio/crossplane/issues/572)
+  * Refresh 0.3 Docs: reflect enhancements, better on-boarding UX, easier to get started[#625](https://github.com/crossplaneio/crossplane/issues/625)
+  * DevOps pipeline examples for Jenkins, GitLab, GitOps using GCP and AWS stacks [#631](https://github.com/crossplaneio/crossplane/issues/631)
+
+## Under Consideration
```

Comment:
can we add the next couple milestones with at least a "theme" or a couple of high level bullets for each?  they don't have to link to specific epics/issues yet, but giving the community a sense of the themes for the next couple upcoming milestones would be awesome to be a bit more forward looking!

Reply:
@jbw976 great idea, how does this look?